packages/protojs/src/staking/v1/authz.ts

```diff
@@ -0,0 +1,105 @@
+import Long from "long";
+import _m0 from "protobufjs/minimal";
+import { Coin } from "../../cosmos/base/v1beta1/coin";
+export declare const protobufPackage = "nibiru.staking.v1";
+/**
+ * AuthorizationType defines the type of staking module authorization type
+ *
+ * Since: cosmos-sdk 0.43
+ */
+export declare enum AuthorizationType {
+    /** AUTHORIZATION_TYPE_UNSPECIFIED - AUTHORIZATION_TYPE_UNSPECIFIED specifies an unknown authorization type */
+    AUTHORIZATION_TYPE_UNSPECIFIED = 0,
+    /** AUTHORIZATION_TYPE_DELEGATE - AUTHORIZATION_TYPE_DELEGATE defines an authorization type for Msg/Delegate */
+    AUTHORIZATION_TYPE_DELEGATE = 1,
+    /** AUTHORIZATION_TYPE_UNDELEGATE - AUTHORIZATION_TYPE_UNDELEGATE defines an authorization type for Msg/Undelegate */
+    AUTHORIZATION_TYPE_UNDELEGATE = 2,
+    /** AUTHORIZATION_TYPE_REDELEGATE - AUTHORIZATION_TYPE_REDELEGATE defines an authorization type for Msg/BeginRedelegate */

```

Comment:
There should be any files committed from `protojs`. The messages just need to be imported in a similar manner to `nibijs/src/msg/bank.ts`: 

```js
import * as pb from "@nibiruchain/protojs/dist/cosmos/bank/v1beta1/tx"
import { Coin } from "@cosmjs/proto-signing"
import { TxMessage } from "./types"

export class BankMsgs {
  /**
   * Creates a 'MsgSend' message to send coins from one account to another.
   *
   * @static
   * @param {string} from - Address of the sender
   * @param {string} to - Address of the receiver
   * @param {Coin[]} coins - Tokens (coins) to transfer from 'from' to 'to'
   * @returns {TxMessage}
   */
  static Send(from: string, to: string, coins: Coin[]): TxMessage {
    return {
      typeUrl: `/${pb.protobufPackage}.MsgSend`,
      value: pb.MsgSend.fromPartial({
        fromAddress: from,
        toAddress: to,
        amount: coins,
      }),
    }
  }
```

You just need to change the import statement:

```js
import * as pb from "@nibiruchain/protojs/dist/cosmos/staking/v1beta1/tx"
```